src/controllers/queryRunner.ts

```diff
@@ -1681,6 +1671,31 @@ export default class QueryRunner {
         return value;
     }
 
+    /**
+     * Vscode core expects uri.fsPath for resourcePath context value.
+     * https://github.com/microsoft/vscode/blob/bb5a3c607b14787009f8e9fadb720beee596133c/src/vs/workbench/common/contextkeys.ts#L275
+     */
+
+    /**
+     * Add query to running queries list
+     * @param ownerUri The owner URI of the query
+     */
+    private addRunningQuery(ownerUri: string): void {
```

Comment:
These two methods (and `updateRunningQueries` below) feel like they ought to be static, or part of a QueryRunnerManager singleton class.  Those methods existing on (each individual) query runner instance feels like it gives the wrong idea of scope-of-impact/responsibility.